src/modules/endpoint/endpoints/categories.js

```diff
@@ -1,53 +1,50 @@
 import EndpointBuilder from '../EndpointBuilder'
 
-import normalizeUrl from 'normalize-url'
 import CategoryModel from '../models/CategoryModel'
 import CategoriesMapModel from '../models/CategoriesMapModel'
 import getCurrentCityConfig from '../../city-detection/getCurrentCityConfig'
+import moment from 'moment'
+import { toPairs } from 'lodash/object'
+import { compose } from 'lodash/fp'
+import normalizePath from 'normalize-path'
 
-function getUrlWithoutCity (url) {
-  const tempUrl = url.split('/')
-  tempUrl.shift()
-  return tempUrl.join('/')
+const PREFIX_LENGTH = 4
+
+// Removes the (unnecessary) path prefix '/:cmsname/:lang/:parent_topic',
+// e.g. '/neuburgschrobenhausenwohnraum/de/raumfrei'
+function getPathWithoutPrefix (path) {
+  if (!path) {
+    return ''
+  }
+  const splitPath = path.split('/').slice(PREFIX_LENGTH)
+  return `/${splitPath.join('/')}`
```

Comment:
I think no template string is required here.

Reply:
Template string is required because of the leading '/'